### ISSUE TITLE
Restructure the "resourcenavbar"

### ISSUE THREAD

**Issue by tinniaru3005** [creativecommons/cc-resource-archive#23]
## Description

The resourcenavbar currently has a very basic look. Upgrade the look & feel of it.

## Reproduction

This can be done in either of 2 ways:
- 1st way:
  - Redisgn it in the form of a drop-down menu
  - We can have a horizontal menubar with 3 options: Topic, medium and Print
  - When we click on each of them we get the list items below
- 2nd way:
  - Add a background behind the headings: Topic, Medium and Print
  - Have a separate div with a proper border for each of them
  - Inside each div, we will have the list items .

## Expectation

This will improve the look and feel of the website

## Screenshots

![1](https://user-images.githubusercontent.com/62856848/221052105-92734585-3aa4-4630-94f3-53ac6ce60cdd.png)

## Resolution

- [x] I would be interested in resolving this bug.

**Comment by kirtanchandak** [comment]
Can I work on this issue ?

**Comment by tinniaru3005** [comment]
Hey, @kirtanchandak welcome to the community thanks for your interest in contributing to the project. But the issue is not yet approved.
The issue will automatically have the status: awaiting triage label applied. Wait for a project maintainer to evaluate the issue and decide whether it's something that they will accept a pull request for.
Once the project maintainer has approved the issue and removed the status: awaiting triage label then a PR can be raised

You can go through the [Contribution Guideline](https://opensource.creativecommons.org/contributing-code/) to know more
Also as I have mentioned in the issue I would like to work on it. So you may look at the other issues.

**Comment by consolexyz** [comment]
Hello ,

**Comment by consolexyz** [comment]
Hello  @possumbilities 
I am interested in working on this issue. From the description and comments, I believe I have the necessary skills and experience to solve this problem.

**Comment by possumbilities** [comment]
I think this would make a good conversation for the community on different approaches we could try to improve this.

**Comment by tinniaru3005** [comment]
That's would be great @possumbilities. 🤩

**Comment by consolexyz** [comment]
okay sounds good

**Comment by KrownWealth** [comment]
I think first way would be a simplify way forward, because it will make the nav-bar more readable, it will reduce junkness, and also give spaces to add a button beside the nav-bar

**Comment by consolexyz** [comment]
well I think a better way to add a hoverable navigation bar , something like the image below 
![image](https://user-images.githubusercontent.com/48781271/229675678-11dab054-4998-4c16-83a4-1cdcb5c2cccb.png)

**Comment by Shivangi-ch** [comment]
I think a good idea will be to add dropdown inputs for each of the three categories(i.e., topic, medium and language). All the current items in the unordered list will be the options for each of the dropdown fields.
When on the home page the dropdowns will have a placeholder value as the category name, and when the user selects a value for any of the three dropdowns they will be redirected to that path.
Basically the same functionality but with dropdowns instead of the unordered lists.

@possumbilities  I would like to work on this issue... also do you have any suggestion on this approach.

**Comment by possumbilities** [comment]
I think there's some merit in exploring a sidebar perhaps. But I don't think hiding the options necessarily makes them more usable. That increases the steps of action from: `landing on page, choose filters as desired`, to: `landing on page, open specific filters area, find filters you want, apply filters, wait for results`.

A sidebar of filters would push the options off to the side, and rearrange the page. A similar UX experience is found here: https://creativecommons.org/blog/archive/ but that works because the filtering isn't a long list. 

There is also another issue with an alternative approach, which is here, I would suggest we move the conversation there: 
* https://github.com/creativecommons/cc-resource-archive/issues/192#issuecomment-1977977045

**Comment by Cappybara12** [comment]
I agree with the proposal to upgrade the look and feel of the resourcenavbar. Adding a dropdown menu structure or enhancing the styling with backgrounds and divs can significantly improve the visual appeal. Additionally, integrating a search functionality within the navbar could enhance user experience by making resource discovery more accessible.

I've already started implementing some of these changes and have a prototype available for review.
![Home_CreativeCommonsResourceArchive-GoogleChrome2024-03-2015-01-08-ezgif com-video-to-gif-converter](https://github.com/creativecommons/cc-resource-archive/assets/122734275/e03da26c-8659-42ba-b332-283c6242dc75)
